tests/test_acasclient.py

```diff
@@ -267,9 +267,65 @@ def get_or_create_global_project():
     output = r.json()
     return output["messages"]
 
+def requires_basic_cmpd_reg_load(func):
+    """
+    Decorator to load the basic cmpdreg data if it is not already loaded
+    """
+    @wraps(func)
```

Comment:
Yeah, I'm not sure best practice.  I knew I wnted to use a decorator and I also found this to be the hardest part of this change mainly because I originally tried to modify self to add some meta about whether the experiment was loaded or not currently (to track the state without having to query the db), but it was failing so I fell back to querying the DB which I think is actually a more robust solution to this.